Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -175,25 +175,62 @@ namespace OpenInfraPlatform
 				{
 					return abs(first - second) < precision;
 				}
-				/*!Compare two 2D vectors for equality to the specified precision
+
+				/*!Compare two doubles for equality to the default precision
+				*\param[in]    first      The first value
+				*\param[in]    second     The second value
+
+				*\return true, if the absolute difference is smaller than the precision. False otherwise
+				*/
+				bool areEqual(const double first, const double second) const
+				{
+					return abs(first - second) < getPrecision();

```

Comment:
```suggestion
					return areEqual(first, second, getPrecision());
```

It is preffered to call other overloads instead of copying & correcting the code.

Reply:
Resolved: 4a44505